app/controllers/concerns/password_helpers.rb

```diff
@@ -19,7 +19,14 @@ def update_password
   end
 
   def deliver_reset_password_instructions(user)
-    user.send_reset_password_instructions
+    token, enc = Devise.token_generator.generate(user.class, :reset_password_token)
+    user.reset_password_token   = enc
+    user.reset_password_sent_at = Time.now.utc
+    user.save(validate: false)
+    message = DeviseMailer.reset_password_instructions(user, token, current_user.name)
+    message.deliver_now
+  rescue => e
+    Rails.logger.error("DEVISE MAILER ERROR: '#{e.message}' while sending reset password mail")
```

Comment:
should we raise the exception to allow sentry to pick it up, or are our logs being alerted on